apps/scorecard/.meteor/.gitignore

```diff
@@ -0,0 +1 @@
+local
```

Comment:
This is definitely redundant and can be removed. There shouldn't be a .gitignore in .meteor

Reply:
molly is dealing with this along with the other gitignores